server.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"net/http"
+	"strings"
+
+	"gopkg.in/juju/charm.v6-unstable"
+
+	"github.com/juju/bundlechanges"
+	"github.com/julienschmidt/httprouter"
+)
+
+func getChangesFromStore(w http.ResponseWriter, r *http.Request, params httprouter.Params) {
+	bundleURL := params.ByName("bundleURL")[1:]
+	fmt.Fprintf(w, "Not implemented; bundle requested: %s", bundleURL)
```

Comment:
Let's return JSON errors, with the proper status code (from net/http). Also, we usually annotate errors using only lowercase strings [shrug].